templates/partials/question.html

```diff
@@ -2,9 +2,13 @@
 {% from "components/fieldset/_macro.njk" import onsFieldset %}
 {% from "components/panel/_macro.njk" import onsPanel %}
 
+{% from 'macros/helpers.html' import interviewer_note %}
+
 {% set form = content.form %}
 
-{% set question_title = question.title %}
+
+{% set title= interviewer_note(question.title) if block.interviewer_only else question.title %}
+{% set question_title= question.title %}
```

Comment:
Not sure why the assignment is made to title, then also to question_title? Can't the reference to question_title be replaced with question.title below?

Reply:
This change was discussed with @ajmaddaford, question_title is used by more than just the question template and in question templates it's used by legend html element so it needs to stay what it was before.

Reply:
`question_title` is used by most of the answer field templates.